src/napari_imagej/_function.py

```diff
@@ -92,26 +99,48 @@
 # Install napari <-> java converters
 when_jvm_starts(lambda: init_napari_converters(ij))
 
+_PTYPE_CONVERTERS = []
 
-# TODO: Move this function to scyjava.convert and/or ij.py.
-def _ptype(java_type):
-    """Returns the Python type associated with the passed java type."""
+def ptype_converter(priority: int) -> Callable[[ModuleItem], Optional[Type]]:
+    def converter(func: Callable[[ModuleItem], Optional[Type]]): 
+        _PTYPE_CONVERTERS.append((func, priority))
+        return func
+    return converter
+
+@ptype_converter(priority = Priority.NORMAL)
+def isAssignableChecker(item: ModuleItem) -> Optional[Type]:
+    java_type = item.getType()
     for jtype, ptype in _ptypes.ptypes.items():
         if jtype.class_.isAssignableFrom(java_type):
             return ptype
+    return None
+
+@ptype_converter(priority = Priority.LOW)
+def canConvertChecker(item: ModuleItem) -> Optional[Type]:
+    java_type = item.getType()
     for jtype, ptype in _ptypes.ptypes.items():
-        if ij.convert().supports(java_type, jtype):
+        if ij.convert().supports(jtype, java_type):
             return ptype
-    raise ValueError(f"Unsupported Java type: {java_type}")
+    return None
+
+
+# TODO: Move this function to scyjava.convert and/or ij.py.
+def _ptype(module_item):
```

Comment:
I decided against this because `_ptype` doeesn't convert types anymore, they convert `ModuleItem`s. `_ptype` is limited to describing equivalent types between Python and Java, making this function beyond that scope. Thus this function has been renamed to `python_type_of`.

This work was done in 2eacbe3